src/main/java/pieselki/bright_utilities/utils/NumericUtils.java

```diff
@@ -0,0 +1,21 @@
+package pieselki.bright_utilities.utils;
+
+import java.text.DecimalFormat;
+
+public class NumericUtils {
+  private static String[] ABBREVATIONS = { "", "K", "M", "B", "t", "q", "Q", "s", "S" };
+
+  public static String getShortNotation(float num) {
+    int abbrevationIndex = 0;
+    while (num > 1000.0f) {
```

Comment:
I don't know Java very well but in JavaScript I will be use `Math.floor(Math.log10(10000000000) / 3) ` to get degree :)

Reply:
It's not easy to keep working on friday afternoon🙈